### DISCUSSION TITLE
New ESP32-S3 boards

### DISCUSSION THREAD

**Discussion by revk** [revk/xESP32-Faikout#98] [Announcements]
My first attempt at the ESP32-S3 version (The ESP32-S1 were out of stock) led to some interference from the regulator, so I am working on new samples with a cleaner design. The first such samples arrived today and I am quite pleased with them, and they work. A key new feature is a medium sized board template to match the Daikin boards. We have plenty of stock of the ESP32-S1 boards still, so will be a while before the S3 are shipping.

Update: Latest are red :-)

![312205](https://github.com/revk/ESP32-Faikin/assets/996983/650071a6-8500-4a89-8cae-14d20da2b37b)
![312206](https://github.com/revk/ESP32-Faikin/assets/996983/4e4a2457-a6f5-4842-a1f8-dc20426a4c2d)

**Comment by ThiefMaster** [comment]
Any idea how long "a while" will be?

Currently it's quite likely that I'll get a Daikin unit at some point this year, so I'll have to decide whether to already order one of the current boards (I guess those are the "will arrive in august 2023" one that are compatible w/ 5V data outputs(?) on newer units?) while they are in stock, or wait until I actually have a data for the unit to be installed and then order either an S1 or S3 model depending on what's available?

**Reply by revk** [reply]
The current stock at Amazon are S1 based, but with the 5V working. It should really make no real difference whether you get the S1 or S3 based ones. It was mainly re-worked for the lack of stock of the S1 modules. The extra PCB cut outs being better is helpful if installing in place of the existing Daikin wifi board in some models, and that is only on the latest ones.

**Reply by ThiefMaster** [reply]
The unit I'm likely getting is a FTXA ("Stylish") which I think is from this or last year.

So probably it makes sense to (at least at some point) disconnect the builtin wifi board... But I guess it's also convenient (at least at the beginning) to have the current (smaller) board which I guess is easier to put somewhere while the official and your wifi board is still there?

**Reply by revk** [reply]
The latest design allows you to break it out to the small PCB as well. You can have around 36mm x 16mm module, or 45mm x 36mm larger PCB (or 70mm x 70mm panel if you really want, LOL).
